### PR TITLE
Support testnet-beta networks

### PR DESCRIPTION
**Pull request by okdas**

### CODE REVIEW

File: .env.sample
Comment (okdas) [inline .env.sample:5]: This will make the network default to Beta TestNet. We would need to update the docs. Should we keep alpha testnet as default instead?

Comment (Olshansk) [reply .env.sample:5]: Beta should be the default

File: docker-compose.yml
Comment (Olshansk) [inline docker-compose.yml:21]: Let's try to upgrade alpha to `0.11` and get everything on the same version.

I know you're working on but just calling it out as a priority to avoid alpha/beta confusion.

**Review by Olshansk — APPROVED** [review]

File: tmp-install-script.sh
Comment (Olshansk) [inline tmp-install-script.sh]: Intentionally merging in `tmp`?

File: docker-compose.yml
Comment (okdas) [reply docker-compose.yml:21]: @Olshansk container image versions have nothing to do with full-node versions anymore, as full-node versions are managed by cosmovisor.
I'm removing an ability to change version to avoid confusion. This will be fully automated after the next cosmovisor release.

File: scripts/init-poktrolld.sh
Comment (Olshansk) [inline scripts/init-poktrolld.sh:9]: Reminder to use TODO_IN_THIS_PR for self comments like this in the future.

Please remove before merging.

**Review by Olshansk — APPROVED** [review]
One minor change but approving otherwise

File: scripts/init-poktrolld.sh
Comment (okdas) [reply scripts/init-poktrolld.sh:9]: Oh, this is not a comment line for ME, but for anybody who wants to test the branch from `pocket-network-genesis` repo.

I thought I'd avoid a `#PUC` with that comment here, haha.

So I would like to keep it. The default branch is `master` of `pocket-network-genesis` repo.

Comment (okdas) [reply scripts/init-poktrolld.sh:9]: I'll merge it with this comment, but if you feel strongly - let me know and I'll remove it.

**Pull request merged**
